docs/concepts/what-is-a-retry-policy.md

```diff
@@ -85,14 +68,35 @@ Non-Retryable Errors = []
 - **Use case**: Use this attribute to ensure that retries do not continue indefinitely.
   However, in the majority of cases, we recommend relying on the Workflow Execution Timeout, in the case of [Workflows](#workflow), or Schedule-To-Close Timeout, in the case of [Activities](#activity), to limit the total duration of retries instead of using this attribute.
 
-### Non-Retryable Errors
+#### Non-Retryable Errors
 
 - **Description**: Specifies errors that shouldn't be retried.
   - **Default is none.**
   - If one of those errors occurs, the [Activity Task Execution](#activity-task-execution) or [Workflow Execution](#workflow-execution) is not retried.
 - **Use case**: There may be errors that you know of that should not trigger a retry.
   In this case you can specify them such that if they occur, the given execution will not be retried.
 
+### Retry interval
+
+The wait time before a retry is the _retry interval_. A retry interval is the smaller of two values:
+
+- The [Initial Interval](#initial-interval) multiplied by the [Backoff Coefficient](#backoff-coefficient) raised to the power of the number of retries.
+- The [Maximum Interval](#maximum-interval).
+
+### Event History
```

Comment:
```suggestion
### Retry Policies and Event Histories
```

Reply:
I'm curious why the repetition here and in `Default values for Retry Policy`. My inclination is toward concision (`Default values` and `Event Histories`), with the idea that the reader knows it's in the context of the Retry Policy page they're reading.

Reply:
For search results.
Event History is a term that could be involved in multiple headers throughout the docs.
I was thinking that this is a great way to differentiate quickly from the results.

Reply:
It looks like if it were just `Default values`, it would still come up in search, since "retry" matches "What is a Retry Policy"

<img width="565" alt="image" src="https://user-images.githubusercontent.com/251288/156948320-93b35be1-1be2-4799-864c-c4e368007539.png">

Reply:
If there is an opportunity to be more specific - I think we should take it.
There is only something to be gained here I should think? Nothing is lost.

Reply:
> If there is an opportunity to be more specific - I think we should take it.

I'd consider 

`Properties > Default values for Retry Policy` header on the Retry Policy page 

to be no more specific than 

`Properties > Default values` header on the Retry Policy page.

In both cases, it's clear to me the answer to "which default values?" is "retry policy properties".

> There is only something to be gained here I should think? Nothing is lost.

I wonder whether "for Retry Policy" would fit under @djmagee 's definition of "filler language"?

Regardless, there's something lost for me, and I imagine some subset of others. My experience of the "for Retry Policy" version is:

- Takes longer to parse/read/understand, or to come back later and glance at and understand, compared to the more concise version below.
- Takes me out of flow a bit—my brain hiccups and is like, "wait, so if this thing is about retry policy, what is Properties and Initial interval about?" Of course the answer is obvious, but my brain for a moment automatically imagines there must be a reason for the qualifier prepositional phrase—that other things around it are less specific, or apply to something else.


<img width="632" alt="image" src="https://user-images.githubusercontent.com/251288/157140297-03f270f4-a5b9-43dd-bf35-239a3ac1938a.png">

<img width="578" alt="image" src="https://user-images.githubusercontent.com/251288/157140364-155025d4-4866-485e-bc75-9a2a03ad1c6d.png">

Reply:
Ya, I think you definitely made an air tight case here - well done.

Reply:
Thanks! ☺️